src/constants/sort-options.constant.ts

```diff
@@ -0,0 +1 @@
+export const sortOptions = ['Price (Low-High)', 'Newest', 'Oldest'] as const;
```

Comment:
Could you please add one more filter option
 
 
![image](https://github.com/jaxin007/fe-react-2024/assets/108006836/fccb8a30-9c19-4a9a-a7cb-3d1d7eba503a)